src/garage/torch/modules/mlp_module.py

```diff
@@ -81,7 +81,9 @@ def forward(self, input_val):
         """Forward method."""
         x = input_val
         for layer in self._layers[:-1]:
-            x = self._hidden_nonlinearity(layer(x))
+            x = layer(x)
+            if self._output_nonlinearity is not None:
```

Comment:
Why is this checked?  I think @lywong92 is also trying to fix the output non-linearity too.

Reply:
I talked with Linda about this, and I think we have some confusion about this part.
I would revert this and wait for Linda's PR

Reply:
I need this changes for the testcases of the GaussianMLPModule.
So I think I would wait until @lywong92 update this code